lib/Page.js

```diff
@@ -62,10 +63,43 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function insertFlexDiv(pageData) {
```

Comment:
`insertFlexDivBeforeFooter` might be more descriptive.

Reply:
Changed behaviour of function to be more like `insertFlexDivBeforeLastFooter`